apps/ejabberd/include/ejabberd.hrl

```diff
@@ -18,8 +18,6 @@
 %%% 02111-1307 USA
 %%%
 %%%----------------------------------------------------------------------
--ifndef(MONGOOSEIM_EJABBERD_HRL).
```

Comment:
We don't have any header files including `ejabberd.hrl`, so when it's used, then it's included by `.erl` file. And including it twice would indicade a mistake or sloppy coding. :)
I think the original reason behind this were habits from C world.